docs/reference_guide.md

```diff
@@ -1223,6 +1247,9 @@ Attaching 2 probes...
 
 This prints the rate of syscalls per second.
 
+Examples in situ:
+[search /tools](https://github.com/search?q=interval%3A+path%3Atools+extension%3Abt&type=Code)
```

Comment:
I think this is searching all of github

Reply:
You are right, I tried playing with the search query and I messed up, will fix